Bonsai.Editor/EditorForm.cs

```diff
@@ -1415,7 +1416,7 @@ void HandleWorkflowError(Exception e)
                     workflowError = workflowException;
                     HighlightExceptionBuilderNode(workflowException, building);
                 }
-                else editorSite.ShowError(e.Message, Name);
+                else editorSite.ShowError(e.Message, "Internal workflow error");
```

Comment:
```suggestion
                else editorSite.ShowError(e.Message, "Unexpected Exception");
```

I would maybe just keep the caption general, since I'm not sure we can guarantee at this point whether these will be only internal errors.